content/blog/2024-05-23-linebender-roadmap.md

```diff
@@ -0,0 +1,248 @@
+- **PR:** [parley#26](https://github.com/linebender/parley/pull/26)
+
+### Software renderer
+
+- **Priority:** Low.
+- **Assigned:** Chad Brokaw.
+- **Description:** Create a software renderer for the glyphs created by Parley, so that users can easily import Parley in their library without depending on a graphics API like Vello. The software renderer should be black-and-white for simplicity. We'll call for volunteers to implement COLRv1 and other advanced features.
+- **PRs:** [parley#54](https://github.com/linebender/parley/pull/54), [parley#55](https://github.com/linebender/parley/pull/55)
+
+
+## Xilem
+
+### Android bringup
+
+- **Priority:** Highest.
+- **Assigned:** Daniel McNab.
+- **Description:** We want Xilem to reliably work on Android. Running your Xilem app should require a small number of steps. Ideally, it should be as simple as running `cargo some-command` in your project.
+- **PR:** [xilem#309](https://github.com/linebender/xilem/pull/309)
+
+### Update repository layout.
+
+- **Priority:** High.
+- **Assigned:** Olivier Faure.
+- **Description:** We've agreed on a layout for Linebender repositories. We should move projects in the Xilem repo to match that layout.
+- **PRs:** [xilem#302](https://github.com/linebender/xilem/pull/302), [vello#590](https://github.com/linebender/vello/pull/590)
+
+### Rework xilem_core to be compatible with Masonry backend
+
+- **Priority:** High.
+- **Assigned:** Daniel McNab.
+- **Description:** The version of Xilem that was ported to Masonry used a different `View` trait that is incompatible with the current `xilem_core` crate. We need to rework `xilem_core` to get traits that can be used by both Masonry and `xilem_web`. Doing so while keeping readable code is non-trivial.
+- **PR:** [xilem#235](https://github.com/linebender/xilem/pull/235), [xilem#310](https://github.com/linebender/xilem/pull/310)
+
+### Write Documentation
+
+- **Priority:** High.
+- **Assigned:** Olivier Faure.
+- **Description:** Xilem's lack of documentation is one of the problems new users complain about the most often. We need to avoid staying in that state for too long.
+- **Issue:** [xilem#392](https://github.com/linebender/xilem/issues/392)
+
+### Re-implement async integration
+
+- **Priority:** Medium.
+- **Assigned:** Daniel McNab.
+- **Description:** One of Xilem's selling point is the ability to handle virtual lists and async sources of data. That ability was temporarily removed for the Masonry port, but we'd like to add it back it. We don't want Masonry to be aware of async stuff, but the AppDriver trait should at least provide a method which lets Xilem access Masonry internals on its own initiative.
+- **Issue:** [xilem#340](https://github.com/linebender/xilem/issues/340)
+
+### Add unit tests
+
+- **Priority:** Low.
+- **Assigned:** Olivier Faure.
+- **Description:** We need to figure out an architecture for doing unit tests in Xilem. Tests can then be added by volunteer contributors.
```

Comment:
I'm not sure that we should "delegate" testing to external contributors; we should aim for it to be part of the process that everyone follows; and hopefully we can make it be very straightforward.